python/tests/plotting/plot_1d_test.py

```diff
@@ -81,7 +89,10 @@ def test_plot_collapse():
 
 
 def test_plot_sliceviewer_with_1d_projection():
-    plot(make_dense_data_array(ndim=3), projection="1d")
+    da = make_dense_data_array(ndim=3)
+    plot(da, projection="1d")
+    plot(da, projection="1d", resampling_mode='sum')
+    plot(da, projection="1d", resampling_mode='mean')
 
 
 def test_plot_sliceviewer_with_1d_projection_with_nans():
```

Comment:
Just noticed that we don't have tests for 1d plots of binned data?

Reply:
:(
Maybe we should have a test that is parametrized on all possible params of `plot()` that tests all possible combinations?